smartcontracts/contracts/OwnableDID.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity >=0.4.22 <0.6.0;
+
+import './DIDBase.sol';
+import './ownership/Ownable.sol';
+
+// OwnableDID is an implementation of IDID which is only updatable by owner
+contract OwnableDID is DIDBase, Ownable {
+    function verifyDID(string memory did) public view returns (bool);
+
+    function create(string memory did, bytes32 hash, string memory uri) public onlyOwner {
```

Comment:
do we need func like this createDIDSigned,so the device can sign a message and no need to call grpc?